packages/functional-tests/pages/products/subscriptionManagement.ts

```diff
@@ -0,0 +1,42 @@
+import { BaseLayout } from '../layout';
+
+export class SubsciptionManagementPage extends BaseLayout {
+  readonly path = '/subscription';
+
+  async cancelSubscription() {
+    return Promise.all([
+      await this.page
+        .locator('[data-testid="reveal-cancel-subscription-button"]')
+        .click(),
+      await this.page
+        .locator('[data-testid="confirm-cancel-subscription-checkbox"]')
+        .click(),
+      await this.page
+        .locator('[data-testid="cancel-subscription-button"]')
+        .click(),
+      await this.page.locator('[data-testid="dialog-dismiss"]').click(),
+    ]);
+  }
+
+  async resubscribe() {
+    return Promise.all([
+      await this.page
+        .locator('[data-testid="reactivate-subscription-button"]')
+        .click(),
+      await this.page
+        .locator('[data-testid="reactivate-subscription-confirm-button"]')
+        .click(),
+      await this.page
+        .locator('[data-testid="reactivate-subscription-success-button"]')
+        .click(),
+    ]);
+  }
+
+  getResubscriptionPrice() {
+    return this.page
+      .locator(
+        '//html/body/div/div/div/div[2]/div/div/div/div/section[3]/div/div/div/div[1]/div[1]'
```

Comment:
😯

Not necessarily in this PR, but might be worth a mild future refactor to add a `data-testid` or `id` or `class` to make this less... scary.